gsk4/src/color_stop.rs

```diff
@@ -2,8 +2,25 @@
 
 use gdk::RGBA;
 use glib::translate::*;
+use std::borrow::Cow;
 use std::fmt;
 
+#[derive(Debug, Default)]
```

Comment:
You should also add a `builder` method to `ColorStop` so you can use it with `ColorStop::builder`

Reply:
wdym? `pub fn builder() -> ColorStopBuilder` also implemented in `ColorStop`.